topics/basics/handling_results/README.md

```diff
@@ -131,10 +131,10 @@ while (has_more_pages) {
   CassFuture* query_future = cass_session_execute(session, statement);
 
   const CassResult* result = cass_future_get_result(query_future);
+  cass_future_free(query_future);
```

Comment:
If the future is freed before the error is handled, then you will not be able to inspect the future to determine the type of error that may have occurred using `cass_future_error_result(query_future)`, `cass_future_error_code(query_future)`, or `cass_future_error_message(query_future)`. I think we should keep the `cass_future_free(query_future)` in the `if` block below the `/* Handle error */` section.

Reply:
Ok, then I'll insert an additional `cass_future_free(query_future);` at the end of the loop